### PR TITLE
Add script for copying prod run

### PR DESCRIPTION
**Pull request by lukebjerring**
Note that this should not be merged until https://github.com/w3c/wptdashboard/issues/241 is resolved (as it includes a local upload-token setup hack to get this to work, for completeness).

### CODE REVIEW

File: test_run_handler.go [hunk 1]
```diff
@@ -37,19 +38,25 @@ func testRunHandler(w http.ResponseWriter, r *http.Request) {
 
 func handlePost(w http.ResponseWriter, r *http.Request) {
 	ctx := appengine.NewContext(r)
-	var err error
 
 	// Fetch pre-uploaded Token entity.
-	suppliedSecret := r.URL.Query().Get("secret")
+  var queryParams url.Values
```
Comment (rwaldron) [inline test_run_handler.go]: Nit: Strange indentation here?

Comment (lukebjerring) [reply test_run_handler.go]: Fixed - I'm not sure which editor is doing that, I bounce between pycharm/gogland/sublime/atom

**Comment by lukebjerring** [comment]
Reverted the secret-adding part, as this is now covered by a python script in https://github.com/w3c/wptdashboard/pull/248 which is safe.

File: test_run_handler.go [hunk 2]
```diff
@@ -67,16 +74,19 @@ func handlePost(w http.ResponseWriter, r *http.Request) {
 	}
 	testRun.CreatedAt = time.Now()
 
-	fmt.Fprintf(w, "got... %v", testRun)
```
Comment (lukebjerring) [inline test_run_handler.go]: Note that this output is (was) a bit _debuggy_. The output is now just the JSON blob as stored in the DB, which is a pretty standard behaviour in a REST API.

**Pull request merged**